fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -612,6 +612,309 @@ void partitionedJoinedIndexTest() {
         }
     }
 
+    @Test
+    void simpleCrossPartitionQuery() {
+        try (FDBRecordContext context = openContext()) {
+            rebuildIndexMetaData(context, COMPLEX_DOC, COMPLEX_PARTITIONED);
+
+            // partition 0 covers 30 days beginning 60 days ago
+            createPartitionMetadata(COMPLEX_PARTITIONED, Tuple.from(1), 0,
+                    Instant.now().minus(60, ChronoUnit.DAYS).toEpochMilli(),
```

Comment:
Yeah, I did consider that, but didn't change it because, like you said, our granularity here is in DAYS, and also calling `Instant.now()` repeatedly gives us (hopefully) slightly different timestamps for the messages.